bin/codacy-analysis-cli.sh

```diff
@@ -31,6 +31,7 @@ run() {
     output_volume="--volume ${OUTPUT_DIRECTORY}:${OUTPUT_DIRECTORY}";
   fi
   local CODACY_ANALYSIS_CLI_VERSION="${CODACY_ANALYSIS_CLI_VERSION:-stable}"
+  docker pull codacy/codacy-analysis-cli:${CODACY_ANALYSIS_CLI_VERSION}
```

Comment:
This seems to work as I tried locally with the changed script to fetch a latest stable version or other version.

I'd reccomened the change that machado said to add quotes, I tried with things like `env CODACY_ANALYSIS_CLI_VERSION="stuff stuff"` and it really gets a little better UX with the quotes.

Also It seems better to have the commands connected with a &&, I just checked locally and seems to behave better in case of a bad tag passed since it doesn't try to run something invalid after already tried to pull something invalid and failling. So:

```suggestion
  docker pull codacy/codacy-analysis-cli:"${CODACY_ANALYSIS_CLI_VERSION}" &&
```

Reply:
Also bellow on line 45:

`codacy/codacy-analysis-cli:${CODACY_ANALYSIS_CLI_VERSION}` should be `codacy/codacy-analysis-cli:"${CODACY_ANALYSIS_CLI_VERSION}"`

So it's consistent.